README.md

```diff
@@ -1,2 +1,58 @@
 # weather-app-Yazeed-Nader
-weather app by  Yazeed El-Haj Salem --- nader shakshak 
+---
+
+## Links
+---
+- [Heroku Server Link](https://weather-app-yazeed-nader-ca12.herokuapp.com/)
+
+---
+
+## Description of the app
+---
+The weather-app provides users with the weather in all countries around the world.
+
+---
+
+## User Journey
+---
+- The user starts typing the city or country in which he wants to see the weather, and then immediately the suggestions begin to appear as soon as he type the name of the city or country
+These suggestions are fetched from the server, as we have a file with all the cities and countries of the world
+And after he chooses the country or city he wants, the site displays the weather for that day and also for seven whole days for this country
+It is also marked on the map for the country or city that he searched for
+
+---
+
+## User Stories
+---
+- Aِs a User I wanna get the weather in the city or country that I like .
+- Aِs a User I would like to get suggestions if I start typing the name of the city .
+- As a User When I start typing the name of the city, I want to get the weather in this city for today .
+- As a User If you write the name of the city or country you will get the weather for a whole week .
+- As a User I would like to see the city I am searching for on the map .
+
+---
+
+
+
+## The way of installation the project to run it locally.
```

Comment:
nice read me, always nice to have. But the installation guide needs more details like install dependencies npm i run start command npm run dev etc